packages/initializer/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
- Name should be `create-stryker`.
- Fields like license, author, version, etc should be in sync with the other packages
- scripts should not contain stuff like `npm install ...`, since stryker-mutator core is already a dependency.
-